### ISSUE TITLE
improved documentation for players

### ISSUE THREAD

**Issue by biblicabeebli** [tlentz/d2modmaker#88]
Specifically, I am noticing that the difference between the Generator and Randomizer modes is not detailed anywhere in the 0.5.2 alpha's ui.

The tooltips could generally be a little more verbose, extend the sentence fragments, and the language should always assume that the user might not know which related options and game objects a checkbox or slider interacts with.  Names of concepts within the context of the project should be emphasized in some way, like by capitalizing them.

Finally, while there is a wiki button on the repo, there is no wiki!

The details provided in the readme should be links to other resources, like the wiki and the releases page, as well as some developer intro comments and a welcome message.
Generally, the wiki should take over documenting what the program _does_ and how to use it, and the readme should state what the project _is_.

By removing the pressure on the readme (which is effectively tied to the state of code in the current master branch) you will have to do less work to maintain that front page, and you can open up editing privileges on the wiki so that the community can contribute.  You can also have a "Features In Development" area of documentation that is subject to change or complete removal, and request user feedback by making submitting issues with a particular tag so that you know they come from potentially non-developers.

**Comment by Amek206** [comment]
Yeah the lack of description of them both is tough, we still are not sure what we are going to do with them. We might end up merging them together in some form. With randomizer being a "chaos" setting within a balanced generator. 

We still have not talked about this enough to know what we want to do. 

A wiki might be something to look into. Thanks!

**Comment by biblicabeebli** [comment]
erm, just in case I'm missing from your last comment there... Github has a built-in wiki feature. You seem to have it enabled for the repo, but when empty it redirects to the repo page.

**Comment by Amek206** [comment]
We just haven't been using it I guess,, I think once we get to a point in the project where we get the new UI and a few more things done we could work on filling out a wiki.
